torchvision/transforms/v2/_misc.py

```diff
@@ -205,6 +205,31 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         return self._call_kernel(F.gaussian_blur, inpt, self.kernel_size, **params)
 
 
+class GaussianNoise(Transform):
+    """Add gaussian noise to the image.
+
+    The input tensor is expected to be in [..., 1 or 3, H, W] format,
+    where ... means it can have an arbitrary number of leading dimensions.
+
+    The input tensor is also expected to be of float dtype in ``[0, 1]``.
+    This transform does not support PIL images.
+
+    Args:
+        mean (float): Mean of the sampled normal distribution. Default is 0.
+        sigma (float): Standard deviation of the sampled normal distribution. Default is 0.1.
+        clip (bool, optional): Whether to clip the values in ``[0, 1]`` after adding noise. Default is True.
+    """
+
+    def __init__(self, mean: float = 0.0, sigma: float = 0.1, clip=True) -> None:
```

Comment:
I think `sigma` (σ) should instead be called `var` for "variance". For example, `mean` is not called `mu` (μ).

Reply:
That's fair @Richienb , I can't claim `mean` and `sigma` are the best name combination, but unfortunately for consistency this is probably what we'll need to use, as these are the names that are already used by `GaussianBlur`.

Reply:
Ok, let's save this for a future breaking change.